logictest/runner.go

```diff
@@ -255,24 +254,18 @@ func (r *Runner) handleInsert(ctx context.Context, c *datadriven.TestData, arrow
 
 	buf.Sort()
 
-	if arrow {
-		converter := pqarrow.NewParquetConverter(memory.NewGoAllocator(), logicalplan.IterOptions{})
-		defer converter.Close()
+	converter := pqarrow.NewParquetConverter(memory.NewGoAllocator(), logicalplan.IterOptions{})
```

Comment:
We're still creating records from parquet buffers here. Maybe change this to construct arrow records directly in this PR or leave that as a TODO.

Reply:
Good call. I'll add an issue and leave it as todo